addons/functions/functions/fnc_deleteTrench.sqf

```diff
@@ -0,0 +1,34 @@
+#include "script_component.hpp"
+/*
+ * Author: nomisum
+ * Safely handles Trench Deletion, manages Variables
+ *
+ * Arguments:
+ * 0: Trench <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [TrenchObj] call grad_trenches_functions_fnc_deleteTrench
+ *
+ * Public: No
+ */
+
+params ["_trench"];
+
+_trench setVariable ["ace_trenches_digging", false, true];
+_trench setVariable [QGVAR(diggingType), nil, true];
+_unit setVariable [QGVAR(diggingTrench), false, true];
+[QGVAR(addDigger), [_trench, _unit, false, true]] call CBA_fnc_serverEvent;
```

Comment:
Where the fuck does `_unit` come from? And why are we addingDiggers in `fnc_deleteTrench`? 😅

Reply:
handleDiggerToGVAR seems to be lost/renamed somewhen into addDigger which is ... wrong